conf/app.json

```
@@ -46,5 +46,8 @@
     "certPath": "",
     "ssoUrl": "",
     "portalUrl": ""
+  },
+  "maps": {
+    "accessToken": "<TOKEN>"
```

Let's keep it empty before modifications:
```suggestion
    "accessToken": ""
```
I will treat such case as a way to access maps without authentication.

---

Ok